utils/include/StrToNumWrapper.cpp

```diff
@@ -0,0 +1,69 @@
+#include "stdafx.h"
+
+#include "StrToNumWrapper.h"
+
+namespace StrToNumWrapper{
+bool TryStoi(const std::string& val, int& var){
+    try{
+        var = std::stoi(val);
+    } catch(...) {
```

Comment:
`...` hmmm, brave XD
Catch or instead of it `const std::exception&`

Reply:
D'oh, forgot to changi this. Thnx, changed to logger's macros, please, check it now)